cider/util/CiderBitUtils.h

```diff
@@ -134,6 +140,7 @@ class CiderBitVector {
     return expect_bits << 3;
   }
 
+  std::shared_ptr<CiderAllocator> allocator_;
```

Comment:
Be careful about member sequence if you using initialization list.

Reply:
copy that